README.md

```diff
@@ -122,6 +123,23 @@ $ ocic pod create --config test/testdata/sandbox_config.json
 ```
 # ocic ctr list
 ```
+#### Recovery from errors
+Information about created pods is stored here `/var/lib/ocid/sandboxes`.
+Currently, this information can get out of sync causing errors such as:
```

Comment:
Right, see also my comment below, I would rather have an issue opened than something in the README - people get worried pretty fast :smile: